### ISSUE TITLE
Value fields in inner nodes of plot_tree don't make sense

### ISSUE THREAD

**Issue by leftys** [microsoft/LightGBM#2755]
Hello, I am using `plot_tree` with LightGBM version 2.3.0 and I was surprised with the value fields in inner nodes (not leaves). The values don't make any sense to me. I would expect the average value in data (or descending leaves) in the value field.

Example:
![lightgbm](https://user-images.githubusercontent.com/1970226/74245753-caaf6180-4ce3-11ea-9347-5e43e5b8c6a0.png)

I would expect the value between 0.0011 and 0.0039, but it is 0.2604.

**Comment by guolinke** [comment]
The leaf output is multiplied to learning rate, while internal node doesn't.
And if there is not even partition, it is not the average.

**Comment by leftys** [comment]
Can such value be useful? Or could it be fixed somehow? Maybe a simple fix could be traverse the tree in the plotting method and re-compute the values of internal nodes?
